CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 Change log
 -----------
 
+* Drop the external initramfs and only use the initramfs bundled in the kernel [Florin]
```

Comment:
That is not correct. Only the builds which provide new versions will be deployed to staging. If you merge now a PR which is not touching VERSION, it will not get deployed to staging. That's why I say you actually released now.